resources/i18n/it_IT/cura.po

```diff
@@ -1437,7 +1311,7 @@ msgstr "Bozza"
 
 msgctxt "@action:inmenu menubar:edit"
 msgid "Drop All Models to buildplate"
-msgstr ""
+msgstr "Rilasciare tutti i modelli sulla piastra di costruzione"
```

Comment:
I suspect that in other places the buildplate is translate with "piano di stampa"